R/reportLCOE.R

```diff
@@ -319,8 +337,8 @@ reportLCOE <- function(gdx, output.type = "both"){
 
  # calculate total ccsinjection cost for all techs
  total_ccsInj_cost <- dimReduce(te_annual_inv_cost[getRegions(te_annual_OMF_cost),getYears(te_annual_OMF_cost),"ccsinje"] +
-                                                     te_annual_OMF_cost[,,"ccsinje"])
- # distribute ccs injection cost over techs
+                                                     te_annual_OMF_cost[,,"ccsinje"] + te_annual_secFuel_cost[,,"ccsinje"])
+ # distribute ccs injection cost over techs => this does not include adjustment cost incurred! => what is the reasoning behind it?
```

Comment:
don't quite understand this comment, do you mean why is adjustment cost not added as a OMF cost? What is the unit of total_ccsInj_cost?

Btw I don't quite get the formula here; shouldn't the total cost = annuitized (investment cost + other fixed costs (OMF, adjCost)) + running cost (energy cost, OMV cost, CO2 tax)... It looks like OMF and energy cost are here, but the others(?)

Reply:
I put this in the commit accidentally. The point is that only ccs injection cost without adjustment cost are distributed across technologies. I just wondered why there is no version that includes adjustment cost as these can be very substantial

Reply:
The general formula for total cost should be (and is):
annuitized (investment cost + adjCost)  + OMF cost (don't need to be annuitized) + running cost (energy cost, OMV cost, CO2 tax).
The ccs cost only has investment cost + OMF cost + now also energy cost
Adjustment cost are missing, that was precisely the point of my comment